src/test/java/org/opentripplanner/routing/street/BicycleRoutingTest.java

```diff
@@ -71,12 +105,14 @@ private static String computePolyline(Graph graph, GenericLocation from, Generic
     var temporaryVertices = new TemporaryVerticesContainer(graph, request);
     RoutingContext routingContext = new RoutingContext(request, graph, temporaryVertices);
 
-    var gpf = new GraphPathFinder(new Router(graph, RouterConfig.DEFAULT, Metrics.globalRegistry));
+    var gpf = new GraphPathFinder(
+      new Router(graph, transitModel, RouterConfig.DEFAULT, Metrics.globalRegistry)
+    );
     var paths = gpf.graphPathFinderEntryPoint(routingContext);
 
     GraphPathToItineraryMapper graphPathToItineraryMapper = new GraphPathToItineraryMapper(
-      graph.getTimeZone(),
-      new AlertToLegMapper(graph.getTransitAlertService()),
+      transitModel.getTimeZone(),
```

Comment:
This test only routes on the street network so doesn't actually need a transit model. You could simply it my just inserting mocks/null where needed.

I've taken the liberty to do that here: https://github.com/leonardehrenfried/OpenTripPlanner/commit/d35395f1cc9f6ae16f66ad4f8ec1e0ca56aeaea0

Reply:
After merging https://github.com/opentripplanner/OpenTripPlanner/pull/4233, I want to move the `AlertToLegMapper` to be applied in the `ItineraryListFilterChain`, similar to what was done for the fare calculations.

Reply:
Test simplified